src/TDockWidget.h

```diff
@@ -33,14 +34,16 @@
 
 // TDockWidget contains helpers for User Windows QDockWidget.
 class TDockWidget : public QDockWidget {
+    Q_OBJECT // Needed so we can use qobject_cast on it!
```

Comment:
```suggestion
    Q_OBJECT
```

Reply:
I wasn't sure that it was obvious - hence the comment.

Reply:
I don't think it needs to be explained, we're using Qt after all and this is a common macro